etna/transforms/math/sklearn.py

```diff
@@ -116,8 +118,8 @@ def _fit(self, df: pd.DataFrame) -> "SklearnTransform":
 
     def fit(self, ts: TSDataset) -> "SklearnTransform":
         """Fit the transform."""
-        self.in_column_regressor = self.in_column in ts.regressors
         super().fit(ts)
+        self.in_column_regressor = [True if col in ts.regressors else False for col in self.in_column]  # type: ignore
```

Comment:
In fact here we can save the list out out_columns, which are created from the regressors like this:
```
self.out_column_regresors = [self._get_column_name(in_column) for in_column in self.in_column if in_column in ts.regressors]
```